guides/common/modules/con_active-directory-with-cross-forest-trust.adoc

```diff
@@ -9,4 +9,4 @@ For more information on cross-forest trusts, see https://access.redhat.com/docum
 endif::[]
 
 From the {Project} point of view, the configuration process is the same as integration with {FreeIPA} server without cross-forest trust configured.
-{ProjectServer} has to be enrolled in the IPM domain and integrated as described in xref:Using_FreeIPA_{context}[].
+{ProjectServer} has to be enrolled in the IdM domain and integrated as described in xref:Using_FreeIPA_{context}[].
```

Comment:
We do this wrong a lot, but IdM is a branded name. Upstream is FreeIPA. We do have `{FreeIPA}`, but that expands to `Red{nbsp}Hat Identity Management`. Perhaps we need a short variant of it.